core/plannernotes.c

```diff
@@ -288,6 +359,12 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 					} else {
 						len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; color: red; float: left;'><b>%s</b></td>",
 							gasname(&gasmix));
+						if (get_he(&lastprintgasmix) > 0) {						// For a trimix gas change, save ICD info if previous cylinder had helium
+							if (isobaric_counterdiffusion(dive, dp, &lastprintgasmix, &newgasmix, &icdvalues))			// Do icd calculations
```

Comment:
Here it has to be `gasmix` instead of `newgasmix`

Reply:
done